_data/simple-icons.json

```diff
@@ -4346,7 +4349,8 @@
         {
             "title": "Ingress",
             "hex": "783CBD",
-            "source": "https://ingress.com/assets/fonts/ingress_icons.woff"
+            "source": "https://ingress.com/assets/fonts/ingress_icons.woff",
```

Comment:
This could be https://ingress.com/ only since the font can be used in their homepage source.